Tests/SpeziViewsTests/SnapshotTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SnapshotTesting
```

Comment:
Nice; cool addition to the repo and what we can test!

Reply:
Yes, pretty cool to test layout. Also very easy to use and create baseline snapshots. Seems to have some widespread use out there 🚀 
Might want to check what other repositories we can use this in.